### PR TITLE
[#873] Upgrade the python packages due to the security vulnerabilities

### LINKED ISSUE #873: Security vulnerabilities: urllib3, ansible, Jinja2 and pyyaml
* ansible - https://nvd.nist.gov/vuln/detail/CVE-2019-3828
* jinja - https://nvd.nist.gov/vuln/detail/CVE-2019-10906
* pyyaml - https://nvd.nist.gov/vuln/detail/CVE-2017-18342
* urllib3 - https://nvd.nist.gov/vuln/detail/CVE-2019-11324

### PR DESCRIPTION
**Pull request by mcdoker18**
this closes #873

### CODE REVIEW

**Comment by mcdoker18** [comment]
bot, test app

**Review by kirillmakhonin — COMMENTED** [review]
@mcdoker18, is default parser of new PyYaml package safe?

File: containers/pipeline-agent/Pipfile [hunk 1]
```diff
@@ -9,7 +9,7 @@ sphinx_rtd_theme = "==0.4.1"
 sphinx-autobuild = "==0.7.1"
 recommonmark = "==0.4.0"
 twine = "==1.11.0"
-ansible = "==2.6.4"
```
Comment (alexander-semenets) [inline containers/pipeline-agent/Pipfile:12]: I guess we can update it to 2.7

**Review by kirillmakhonin — APPROVED** [review]

File: containers/pipeline-agent/Pipfile (hunk 1 above)
Comment (mcdoker18) [reply containers/pipeline-agent/Pipfile:12]: Why?

Comment (alexander-semenets) [reply containers/pipeline-agent/Pipfile:12]: we use ansible 2.7.2 for playbooks installation, you can check in ansible container. So maybe that make sense to use the same version across all components, by the way it's not strictly necessary so up to you.

**Review by alexander-semenets — APPROVED** [review]

**Pull request merged**